code/__HELPERS/pronouns.dm

```diff
@@ -201,65 +201,65 @@
 
 //humans need special handling, because they can have their gender hidden
 /mob/living/carbon/human/p_they(capitalized, temp_gender)
-	var/list/obscured = check_obscured_slots()
+	var/obscured = check_obscured_slots()
 	var/skipface = (wear_mask && (wear_mask.flags_inv & HIDEFACE)) || (head && (head.flags_inv & HIDEFACE))
-	if((ITEM_SLOT_ICLOTHING in obscured) && skipface)
+	if((ITEM_SLOT_ICLOTHING & obscured) && skipface)
```

Comment:
```suggestion
	if((obscured & ITEM_SLOT_ICLOTHING) && skipface)
```

Could you change the expressions to have the flag on the right side? It'll work either way but pretty much every use of flags in the code-base does it this way and I believe it's more readable because of that.

Reply:
ah yes that makes sense, I'll get to it